lib/syskit/dynamic_port_binding.rb

```diff
@@ -203,6 +203,9 @@ def initialize(
             # Method called by {Accessor} to create the accessor object from a
             # port
             def create_accessor(port)
+                if port.model.init_policy?
+                    policy = policy.merge(init: port.model.init_policy)
+                end
```

Comment:
I don't remember how this was ... but

- it should fail with a nomethoderror (`policy` is nil in there), as in `policy.merge` refers to the local variable and not the method from the accessor.
- it overrides the explicit policy with the default, and it should be the other way around

You can use `@policy` to refer to the policy given to the constructor.